report.py

```diff
@@ -182,42 +182,44 @@ def run_report(config, blockers, server, header, test, save):
 		summary=summary
 	)
 
-	# parse list of email addresses
-	if test:
-		recipients = config['email_to_test'].split(',')
-	else:
-		recipients = config['email_to'].split(',')
+	# save HTML report to file
+	generate_html_file(htmlcode)
+	print('HTML file generated in "archives" folder')
```

Comment:
nit: folder name appears to be "archive"
```suggestion
	print('HTML file generated in "archive" folder')
```